### PR TITLE
Add docs for supported versions in modules

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: metricbeat/docs/modules/mysql.asciidoc [hunk 1]
```diff
@@ -8,6 +8,10 @@ This file is generated! See scripts/docs_collector.py
 This modules periodically fetches metrics from https://www.mysql.com/[MySQL]
 servers.
 
+The apache MetricSets were tested with 2.4.20 and are expected to work with all version
+>= 2.2.31 and >= 2.4.16.
+
+
```
Comment (radoondas) [inline metricbeat/docs/modules/mysql.asciidoc]: copy/paste from apache?

Comment (ruflin) [inline metricbeat/docs/modules/mysql.asciidoc]: argh :-( thanks for spotting this one.

Comment (ruflin) [inline metricbeat/docs/modules/mysql.asciidoc]: New version pushed.

**Comment by andrewkroh** [comment]
LGTM

**Pull request merged**